lua/rocks/operations.lua

```diff
@@ -145,7 +150,14 @@ operations.sync = function(user_rocks)
                 text:render_char(split.bufnr, -1, linenr_copy, 0, linenr_copy, msg_length)
 
                 table.insert(actions, function()
-                    local ret = operations.install(user_rocks[key].name, user_rocks[key].version).wait()
+                    -- If the plugin version is a development release then we pass `dev` as the version to the install function
+                    -- as it gets converted to the `--dev` flag on there, allowing luarocks to pull the `scm-1` rockspec manifest
+                    local ret
+                    if user_rocks[key].version == "scm-1" then
```

Comment:
```suggestion
                    if vim.startswitn(user_rocks[key].version, "scm-") then
```

Reply:
Oops, typo on my end. Should have been "startswith"

Reply:
fixed :p